target/linux/generic/config-5.10

```diff
@@ -939,6 +939,7 @@ CONFIG_CMDLINE=""
 CONFIG_COMPACTION=y
 # CONFIG_COMPAL_LAPTOP is not set
 # CONFIG_COMPAT is not set
+# CONFIG_COMPAT_32BIT_TIME is not set
```

Comment:
Don't think this should be here. CONFIG_COMPAT_32BIT_TIME is currently set per target, with the default depending on the arch being 32- or 64-bit. In the future, once all userspace and libc support 64-bit time_t, this might make sense though.

Reply:
Thanks for the suggestion. CONFIG_COMPAT_32BIT_TIME has been removed from generic config.